src/components/common/download/download.ts

```diff
@@ -1,15 +1,22 @@
 /*
- * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
  *
  * SPDX-License-Identifier: AGPL-3.0-only
  */
 
+/**
+ * Generate a download link for a given {@link BlobPart file}.
+ *
+ * @param data The file to download.
+ * @param fileName Which filename does the file have.
+ * @param mimeType What is the files mimetype.
+ */
 export const download = (data: BlobPart, fileName: string, mimeType: string): void => {
   const file = new Blob([data], { type: mimeType })
   downloadLink(URL.createObjectURL(file), fileName)
 }
 
-export const downloadLink = (url: string, fileName: string): void => {
+const downloadLink = (url: string, fileName: string): void => {
```

Comment:
Docs missing

Reply:
No I removed the export therefore it's not a public method and we don't document it normally.